.travis.yml

```diff
@@ -1,41 +1,43 @@
 language: php
 
-php:
-  - 7.1
-  - 7.2
-
 env:
   global:
-    - setup=basic
+    - SETUP=stable
 
 matrix:
+  fast_finish: true
   include:
-    - php: 7.1
-      env: setup=lowest
-    - php: 7.1
-      env: setup=stable
     - php: 7.2
-      env: setup=lowest
     - php: 7.2
-      env: setup=stable
+      env: SETUP=lowest
+    - php: 7.3
+    - php: 7.3
+      env: SETUP=lowest
 
 sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
 
+services:
+  - memcached
+  - redis-server
+  - mysql
+
 before_install:
+  - phpenv config-rm xdebug.ini || true
```

Comment:
I'm confused; this always evaluates to "true"

Reply:
Effectively, don't care if is error, but this is from Laravel's travis config.